src/widget/passwordedit.cpp

```diff
@@ -42,10 +42,11 @@ void PasswordEdit::registerHandler()
 void PasswordEdit::unregisterHandler()
 {
 #ifdef ENABLE_CAPSLOCK_INDICATOR
-    if (eventHandler && eventHandler->actions.contains(action))
+    int idx;
+
+    if (eventHandler && (idx = eventHandler->actions.indexOf(action)) >= 0)
```

Comment:
FWIW, it would look like this:
```diff
diff --git a/src/widget/passwordedit.cpp b/src/widget/passwordedit.cpp
index bbcc42b..9757082 100644
--- a/src/widget/passwordedit.cpp
+++ b/src/widget/passwordedit.cpp
@@ -42,15 +42,18 @@ void PasswordEdit::registerHandler()
 void PasswordEdit::unregisterHandler()
 {
 #ifdef ENABLE_CAPSLOCK_INDICATOR
-    int idx;
-
-    if (eventHandler && (idx = eventHandler->actions.indexOf(action)) >= 0)
+    if (eventHandler)
     {
-        eventHandler->actions.remove(idx);
-        if (eventHandler->actions.isEmpty())
+        int idx = eventHandler->actions.indexOf(action);
+
+        if (idx >= 0)
         {
-            delete eventHandler;
-            eventHandler = nullptr;
+            eventHandler->actions.remove(idx);
+            if (eventHandler->actions.isEmpty())
+            {
+                delete eventHandler;
+                eventHandler = nullptr;
+            }
         }
     }
 #endif
```